gapis/api/vulkan/transform_command_disabler.go

```diff
@@ -314,10 +370,10 @@ func (disablerTransform *commandDisabler) getNewCommandBufferAndBegin(ctx contex
 	}
 
 	beginInfo := NewVkCommandBufferBeginInfo(inputState.Arena,
-		VkStructureType_VK_STRUCTURE_TYPE_COMMAND_BUFFER_BEGIN_INFO, // sType
-		NewVoidᶜᵖ(memory.Nullptr),                                   // pNext
-		0,                                                           // flags
-		NewVkCommandBufferInheritanceInfoᶜᵖ(memory.Nullptr), // pInheritanceInfo
+		VkStructureType_VK_STRUCTURE_TYPE_COMMAND_BUFFER_BEGIN_INFO,                                         // sType
+		NewVoidᶜᵖ(memory.Nullptr),                                                                           // pNext
+		VkCommandBufferUsageFlags(VkCommandBufferUsageFlagBits_VK_COMMAND_BUFFER_USAGE_ONE_TIME_SUBMIT_BIT), // flags
```

Comment:
@AWoloszyn @pmuetschard Does this affect performance characteristics?

Reply:
Yes it can. PROBABLY not gpu performance, but it  maybe.

Reply:
Shouldn't this be `existingCommandBuffer.Flags()` ?